internal/server/delete_user_test.go

```diff
@@ -71,16 +71,17 @@ func TestGetDeleteUser(t *testing.T) {
 		User: client.user.data,
 	}, template.lastVars)
 }
-
+/*
```

Comment:
You will want this test, it is failing because the `deleteUser` handler doesn't do any permissions checks. You'll need something like https://github.com/ministryofjustice/opg-sirius-user-management/pull/68/files#diff-57ba42600934eb45316e1d644b6f626fbad60f65a33bbc8068aa9c0132fdc123R30, but with the correct permission